tests/tdastro/effects/test_redshift.py

```diff
@@ -1,71 +0,0 @@
-import numpy as np
-from tdastro.effects.redshift import Redshift
-from tdastro.sources.step_source import StepSource
-
-
-def get_no_effect_and_redshifted_values(times, wavelengths, t0, t1, brightness, redshift) -> tuple:
-    """Get the values for a source with no effects and a redshifted source."""
-    model_no_effects = StepSource(brightness=brightness, t0=t0, t1=t1)
-    model_redshift = StepSource(brightness=brightness, t0=t0, t1=t1, redshift=redshift)
-    model_redshift.add_effect(Redshift(redshift=model_redshift, t0=model_redshift))
-
-    values_no_effects = model_no_effects.evaluate(times, wavelengths)
-    values_redshift = model_redshift.evaluate(times, wavelengths)
-
-    # Check shape of output is as expected
-    assert values_no_effects.shape == (len(times), len(wavelengths))
-    assert values_redshift.shape == (len(times), len(wavelengths))
-
-    return values_no_effects, values_redshift
-
-
-def test_redshift() -> None:
-    """Test that we can create a Redshift object and it gives us values as expected."""
```

Comment:
Is it worth preserving this test? I'm guessing not, because the `test_redshift_values()` looks more comprehensive. But I wanted to check.

Reply:
It felt redundant to me--I think I needed this when I was initially figuring out redshift, but I wouldn't say this offers anything novel (and so is misleading to include it).